### PR TITLE
[Fix] type ModelData.instance_prompt

### PR DESCRIPTION
**Pull request by mishig25**
Follow up to https://github.com/huggingface/huggingface.js/pull/909

### CODE REVIEW

File: packages/tasks/src/model-data.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ export interface ModelData {
 					parameters?: Record<string, unknown>;
 			  };
 		base_model?: string | string[];
-		instance_prompt?: string;
+		instance_prompt?: string | null;
```
Comment (pcuenca) [inline packages/tasks/src/model-data.ts:110]: Interesting, I would have thought that the trailing `?` would imply that.

Comment (mishig25) [reply packages/tasks/src/model-data.ts:110]: `instance_prompt?: string` is `string | undefined`
`instance_prompt?: string | null` is `string | null | undefined`

in the strictest sense `null !== undefined`

Comment (Wauplin) [reply packages/tasks/src/model-data.ts:110]: TIL

Comment (julien-c) [reply packages/tasks/src/model-data.ts:110]: ok when can it be set to `null`?

Comment (mishig25) [reply packages/tasks/src/model-data.ts:110]: I'm not sure but that is how it was defined in moon:

`instance_prompt?: string | null;`

**Pull request merged**